core/dbt/config/selectors.py

```diff
@@ -3,7 +3,7 @@
 from dbt.clients.yaml_helper import yaml, Loader, Dumper, load_yaml_text  # noqa: F401
 from dbt.dataclass_schema import ValidationError
 
-from .renderer import SelectorRenderer
+from .renderer import BaseRenderer
```

Comment:
Makes sense to just use `BaseRenderer`, since the context available to `selectors.yml` is the same as every other yaml files